addons/mrp/models/mrp_workorder.py

```diff
@@ -283,15 +285,15 @@ def _assign_default_final_lot_id(self):
     def record_production(self):
         self.ensure_one()
         if self.qty_producing <= 0:
-            raise UserError(_('Please set the quantity you produced in the Current Qty field. It can not be 0!'))
+            raise UserError(_('Please set the quantity you are currently producing. It should be different from zero.'))
 
         if (self.production_id.product_id.tracking != 'none') and not self.final_lot_id:
             raise UserError(_('You should provide a lot/serial number for the final product'))
 
         # Update quantities done on each raw material line
         raw_moves = self.move_raw_ids.filtered(lambda x: (x.has_tracking == 'none') and (x.state not in ('done', 'cancel')) and x.bom_line_id)
         for move in raw_moves:
-            if move.unit_factor:
+            if move.unit_factor and not move.move_line_ids.filtered(lambda m: not m.done_wo):
```

Comment:
and not move.move_line_ids in self.active_move_line_ids

Reply:
We take care of only treating move lines that aren't in `self.active_move_line_ids` (the wokorder tablet view allows to record specific quantity consumed on non-tracked move).